packages/config/config/devices/0x0438/4512757.json

```
@@ -0,0 +1,534 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "4512757",
+	"description": "16A zwave thermostat - White",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0108"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "4",
+			"label": "ECO Mode",
+			"description": "Allowable range: 0-1",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "8",
+			"label": "Window check",
```

Param labels should be **Title Case**
```suggestion
			"label": "Window Check",
```

---

Corrected these throughout the file